packages/api/tests/endpoints/async-operations/test-create-async-operations.js

```diff
@@ -232,7 +231,7 @@ test('POST returns a 409 error if the async operation already exists in PostgreS
 
 test.serial('POST returns a 500 response if record creation throws unexpected error', async (t) => {
   const { jwtAuthToken } = t.context;
-  const stub = sinon.stub(AsyncOperationModel.prototype, 'create')
```

Comment:
Still seeing 8 references in this file to `AsyncOperationModel`, including import, test setup, and a few other places.

Reply:
Yikes!  Thanks for catching that, doubly so if this was just an omission and not a merge/update situation 😦  df249b9dcc